docs/multicluster/quick-start.md

```diff
@@ -313,8 +314,8 @@ kubectl annotate node node-b1 multicluster.antrea.io/gateway=true
 ### Add new member clusters
 
 If you want to add a new member cluster to your ClusterSet, you can follow the
-steps for cluster B to do so. Remember to update the member cluster ID in
-`member-clusterset-template.yml` to the new member cluster's ID in the step 2 of
+steps for cluster B to do so. Remember to update the member cluster ID `spec.clusterID`
+in `member-clusterset-template.yml` to the new member cluster's ID in step 2 of
```

Comment:
Sorry, when reading this sentence again, I feel we should keep the original "the". Let me revert the change when merging the PR.